src/java/org/apache/cassandra/db/ColumnFamilyStore.java

```diff
@@ -3322,8 +3326,10 @@ public boolean getNeverPurgeTombstones()
     void onTableDropped()
     {
         indexManager.markAllIndexesRemoved();
+        logger.debug("CFS is being dropped: indexes removed");
```

Comment:
would be useful when you include the name of CFS which being dropped (same for the other log message in this file)